### PR TITLE
Merging PRL-460 and PRL-461 changes

### PR DESCRIPTION
**Pull request by shashisk97**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-460
https://tools.hmcts.net/jira/browse/PRL-461 

### Change description ###

Child and applicant details event

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by yogendraU — APPROVED** [review]

File: definitions/private-law/json/CaseEvent.json [hunk 1]
```diff
@@ -89,5 +89,31 @@
...
     "ShowSummary": "Y",
     "ShowEventNotes": "N"
+  },
+  {
+    "LiveFrom": "05/11/2021",
+    "CaseTypeID": "C100",
+    "ID": "applicantsDetails",
+    "Name": "Applicant details",
+    "Description": "Details of the applicants in the case",
+    "DisplayOrder": 88,
```
Comment (pete-lowe) [inline definitions/private-law/json/CaseEvent.json:99]: This should be 8

File: definitions/private-law/json/CaseEvent.json [hunk 2]
```diff
@@ -89,5 +89,31 @@
...
+    "ShowSummary": "Y",
+    "ShowEventNotes": "N"
+  },
+  {
+    "LiveFrom": "05/11/2021",
+    "CaseTypeID": "C100",
+    "ID": "childDetails",
+    "Name": "Child details",
+    "Description": "Details of the children in the case",
+    "DisplayOrder": 89,
```
Comment (pete-lowe) [inline definitions/private-law/json/CaseEvent.json:112]: This should be 9

Comment (yogendraU) [reply definitions/private-law/json/CaseEvent.json:112]: Once all events are done, we will reorder this. it was intentionally added with higher number

File: definitions/private-law/json/CaseEvent.json (hunk 1 above)
Comment (yogendraU) [reply definitions/private-law/json/CaseEvent.json:99]: Once all events are done, we will reorder this. it was intentionally added with higher number

**Review by pete-lowe — APPROVED** [review]

**Pull request merged**